### PR TITLE
fix additional tags in arrangement 6

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: osbs/build/user_params.py [hunk 1]
```diff
@@ -155,10 +157,11 @@ def set_params(self,
...
         self.compose_ids.value = compose_ids or []
 
-        if additional_tag_data:
-            self.additional_tags = AdditionalTagsConfig(additional_tag_data['dir_path'],
-                                                        additional_tag_data['file_name'],
-                                                        additional_tag_data['tags'])
+        if self._repo_info.additional_tags.from_container_yaml:
+            self.tags_from_container_yaml.value = True
+        else:
+            self.tags_from_container_yaml.value = False
```
Comment (twaugh) [inline osbs/build/user_params.py]: `self.tags_from_container_yaml.value = self._repo_info.additional_tags.from_container_yaml`?

**Comment by rcerven** [comment]
yeah that would be possible to creae repo_info in pluginsConfiguration

**Comment by rcerven** [comment]
reworked ready for review again

**Comment by twaugh** [comment]
Not update release notes - fix to unreleased feature.

**Pull request merged**